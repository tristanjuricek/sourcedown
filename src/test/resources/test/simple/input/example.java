/**
 * OMG I'm a JAVA file in a default package.
 */

public class Foo {
    
    public void doNothing() {
        System.out.println("OMG SIDE EFFECT ABORT ABORT");
    }
}