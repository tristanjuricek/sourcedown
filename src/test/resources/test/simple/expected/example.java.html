<!DOCTYPE html>
<html lang="en">
<head>
  <title>example.java</title>
  <meta http-equiv="content-type" content="text/html; charset=UTF-8">
  <link rel="stylesheet" href="./res/style.css"></style>
  <script type="text/javascript" 
    src="./res/jquery.js"></script>
</head>
<body>
  <div id="container">
    <h1 class="title">example.java&nbsp;<span class="change-link">Change</span></h1>
    <ul id="nav-list" class="hidden">
                    
                <li><a href="./dir/example.scala.html">dir/example.scala</a></li>
                                
                <li class="active">example.java</li>
                                
                <li><a href="./example.txt.html">example.txt</a></li>
                        </ul>
    <table><tbody><tr><td><p>OMG I'm a JAVA file in a default package.
</p></td><td><div class="highlight"><pre><span class="kd">public</span> <span class="kd">class</span> <span class="nc">Foo</span> <span class="o">{</span>
    
    
</pre></div>
</td></tr><tr><td><p>This is an example of a one-line comment
with two lines.</p></td><td><div class="highlight"><pre>    <span class="kd">public</span> <span class="kt">void</span> <span class="nf">doNothing</span><span class="o">()</span> <span class="o">{</span>
        <span class="n">System</span><span class="o">.</span><span class="na">out</span><span class="o">.</span><span class="na">println</span><span class="o">(</span><span class="s">&quot;OMG SIDE EFFECT ABORT ABORT&quot;</span><span class="o">);</span>
    <span class="o">}</span>
<span class="o">}</span>
</pre></div>
</td></tr></tbody></table>
  </div>
<script type="text/javascript">
$(function() {
    var $navList = $("#nav-list"),
        $change = $(".title .change-link");
    
    $change.click(function() { $navList.toggleClass("hidden") });
});
</script>
</body>
</html>
